docs/auth-modes.md

```diff
@@ -12,196 +12,26 @@ A combination of Connection Dialog and Connection Resolver changes implement eac
 
 ## Supported Authentication Modes
 
-### No Authentication
-User is never prompted for credentials.
+| Name | value | Description |
+| - | - | - |
+| No Authentication | `auth-none` | User is never prompted for credentials |
+| Username only | `auth-user` | User is prompted for username during initial connection creation. |
+| Username and Password | `auth-user-pass` | User is prompted for username and password during initial connection creation, and password only when reconnecting to the data source |
+| Password only | `auth-pass` | User is prompted for password during initial connection creation and reconnecting to the data source |
+| OAuth | `oauth` | User is prompted with the default brower for [OAuth]({{ site.baseurl }}/docs/oauth) credentials during initial connection creation and reconnecting to the data source |
```

Comment:
We should document `auth-integrated` here as well. 

Also, should we integrate the Kerberos documentation found in [this sample](https://github.com/tableau/connector-plugin-sdk/tree/master/samples/scenarios/jdbc_kerberos) to this doc page? That's so buried I didn't know it existed until Devansh pointed it out to me.